src/core.jl

```diff
@@ -42,26 +42,27 @@ function read_record(io::IO)
 end
 
 """
-    read([f=identity], s::Union{String,Vector{String}};kwargs...)
+    read(s::Union{String,Vector{String}};kwargs...)
 
 Read tensorflow records from file(s).
 
 # Keyword Arguments
 
+- `f=Example`. Changes the type of the elements in the channel to the type specified by `f` if provided.
 - `compression=nothing`. No compression by default. Optional values are `:zlib` and `:gzip`.
 - `bufsize=10*1024*1024`. Set the buffer size of internal `BufferedOutputStream`. The default value is `10M`. Suggested value is between `1M`~`100M`.
 - `channel_size=1000`. The number of pre-fetched elements.
-- `eltype=Example`. Change it to the type of result `f(::Example)` if `f` is provided.
+- `record_type=Example`. The type of value being read from the `file/files` that are provided.
 
 !!! note
 
     To enable reading records from multiple files concurrently, remember to set the number of threads correctly (See [JULIA_NUM_THREADS](https://docs.julialang.org/en/v1/manual/environment-variables/#JULIA_NUM_THREADS)).
 """
-read(s; kw...) = read(identity, s; kw...)
+read(s; kw...) = read(s; kw...)
```

Comment:
I think you might have some misunderstanding here.

The data type read from tensorflow record is always `Example`. So we don't need to specify it explicitly. Here the `record_type` means the datatype of the result `f(x::Example)`.

Reply:
Ahh, I see.